_source/user-guide/infrastructure-monitoring/grafana-explore-p8s.md

```diff
@@ -72,8 +85,8 @@ You can click the <i class="fas fa-signal"></i> icon for a top-N analysis of the
 
 
 ### Split and Sync
-
-You can split your Explore mode screen to work faster and make quick comparisons. You can split the screen to work with two views side-by-side, whether Metrics-Metrics, Logs-Logs, or Logs-Metrics.
+<!-- still relevant-->
+You can split your Explore mode screen to work faster and make quick comparisons. You can split the screen to work with two views side-by-side, whether Metrics-Metrics, Logs-Logs, or Logs-Metrics.  <!-- what sort of things would you display in the 2 panels, for example?  -->
```

Comment:
True - this one's still relevant.
It's especially effective for checking how queries behave in different time ranges and even different data sources (in Logz.io world, data source  = a different metrics account), so we can show something like that